pkg/constants/constants.go

```diff
@@ -56,6 +56,7 @@ const (
 	LocalDirBuildContextPrefix = "dir://"
 	GitBuildContextPrefix      = "git://"
 	HTTPSBuildContextPrefix    = "https://"
+	FileBuildContextPrefix     = "file://"
```

Comment:
move this to buildcontext package since its only used there. 
Not your fault since kaniko has all constants bundled in constants pkg.